addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -48,9 +48,8 @@
             <field name="arch" type="xml">
                 <form string="Mailing">
                     <header style="min-height:31px;">
-                        <button name="action_put_in_queue" type="object" attrs="{'invisible': [('state', 'in', ('in_queue', 'done'))]}" class="oe_highlight" string="Send"
+                        <button name="action_put_in_queue" type="object" attrs="{'invisible': [('state', 'in', ('in_queue', 'done'))]}" class="oe_highlight" string="Launch"
```

Comment:
Not sure it's intended to get the confirmation: "This will send the email to all recipients. Do you still want to proceed ?" when you are scheduling it for later.

Maybe we will need 2 buttons that call the same action, but ask confirmation based on the value of "schedule_type".